packages/quantic/cypress/integration/sort/sort.cypress.ts

```diff
@@ -0,0 +1,111 @@
+import {configure} from '../../page-objects/configurator';
+import {
+  InterceptAliases,
+  interceptSearch,
+  interceptSearchIndefinitely,
+} from '../../page-objects/search';
+import {SortExpectations as Expect} from './sort-expectations';
+import {SortActions as Actions} from './sort-actions';
+import {SearchExpectations} from '../search-expectations';
+
+describe('quantic-sort', () => {
+  const sortUrl = 's/quantic-sort';
+
+  const sortOptions = ['relevancy', 'date descending', 'date ascending'];
+  const sortOptionLabels = ['Relevancy', 'Newest', 'Oldest'];
+
+  function visitSort(waitForSearch = true) {
+    interceptSearch();
+    cy.visit(sortUrl);
+    configure();
+    if (waitForSearch) {
+      cy.wait(InterceptAliases.Search);
+    }
+  }
+
+  function loadFromUrlHash(urlHash: string) {
+    interceptSearch();
+    cy.visit(`${sortUrl}#${urlHash}`);
+    configure();
+  }
+
+  function setupWithPauseBeforeSearch() {
+    interceptSearchIndefinitely();
+    cy.visit(sortUrl);
+    configure();
+  }
+
+  it('should not render before results have returned', () => {
+    setupWithPauseBeforeSearch();
+
+    Expect.displaySortDropdown(false);
+  });
+
+  describe('when loading default sort', () => {
+    it('should work as expected', () => {
+      visitSort();
```

Comment:
I'd suggest to start the test like this:

    visitSort(false);

    Expect.displaySortDropdown(false);
    cy.wait(InterceptAliases.Search);

    Expect.displaySortDropdown(true);
    ...

This way, we should be able to remove the test `should not render before results have returned`.